LayoutTest/TestCase/LYTLayoutTestCase.m

```diff
@@ -29,6 +31,16 @@ @implementation LYTLayoutTestCase
 
 #pragma mark - Running Tests
 
++ (void)setUp {
```

Comment:
It looks like you're overriding the class function here. I didn't know this even exists...I assume this is called once before all tests in this class are run? While the instance method gets run before each test?